litmus-portal/frontend/src/components/WorkflowCard/styles.ts

```diff
@@ -42,16 +42,16 @@ const useStyles = makeStyles((theme) => ({
     margin: theme.spacing(1),
     textAlign: 'center',
     cursor: 'pointer',
-    border: `1px solid ${theme.palette.text.hint}`,
```

Comment:
Can we replace `px` with some other relative unit

Reply:
yes @whitetiger1399 , I have just fixed the colours as these components are being removed as part of the v2 redesign so increasing the number of changes with this, won't be of much use here. We may refactor the code while redesigning which is already under progress on a separate branch.

Reply:
For border px is fine...you can change but it's fine...too small a thing... you won't notice it being responsive.